api/src/core/sql/schema/v1.schema.sql

```diff
@@ -299,9 +299,6 @@ create table word_tags_votes(
 create table phrases(
   phrase_id bigserial primary key,
   words bigint[] not null, -- references words(word_id)
```

Comment:
In my opinion, even though phrase is consists of words, I think it is better to store phraselike_string just like word

once we define phrase, then searchability is very important, if we doesn't save we need to do more calcuation.